pina/data/dataset.py

```diff
@@ -248,6 +208,36 @@ def create_graph_batch(self, data):
         """
         # TODO
         """
-        if isinstance(data[0], Data):
-            return self._create_graph_batch_from_list(data)
-        return self._create_output_batch(data)
+        return self._create_graph_batch_from_list(data)
+
+    @staticmethod
+    def _extract_output(func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            out = func(*args, **kwargs)
+            batch = kwargs["batch"]
+            # Copying y into ouput_points
+            out["output_points"] = batch.y
+            # Deleting y from batch
+            batch.y = None
+            # Store new batch withou y
+            out["input_points"] = batch
+            return out
+
+        return wrapper
+
+    @staticmethod
+    def _extract_cond_vars(func):
```

Comment:
I understand `_extract_output` and why it exists, but what is the purpose of `_extract_cond_vars`?